obal/data/roles/setup-sources/tasks/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+---
+force: false
+annex_opts: ""
```

Comment:
Convention says to prefix variables with the role to namespace them and prevent clashes.